src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -154,28 +154,29 @@ public PersistentDataStore() {
    * Loads all Activity objects from the Datastore service and returns them in a List, sorted in
    * ascending order by creation time.
    * @throws PersistentDataStoreException if an error was detected during the load from the
-   *			Datastore service
+   *      Datastore service
    */
    
   public List<Activity> loadActivities() throws PersistentDataStoreException {
-  List<Activity> activities = new ArrayList();
+  List<Activity> activities = new ArrayList<>();
   
-  	// Retrieve all activities from the datastore.
-  	Query query = new Query("chat-activities").addSort("creation_time", SortDirection.ASCENDING);
-  	PreparedQuery results = datastore.prepare(query);
+    // Retrieve all activities from the datastore.
+    Query query = new Query("chat-activities").addSort("creation_time", SortDirection.ASCENDING);
```

Comment:
So here you are getting all chat activities and sorting them right. What I'm curious about is where is this addSort method coming from, does it come with Query?

Reply:
Yepp, queries typically have three properties: the entity kind, sort order, and filters for the entities. If you want to read up on it, the Google Cloudstore documentation is pretty helpful: https://cloud.google.com/datastore/docs/concepts/queries#datastore-datastore-basic-query-java